packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.tsx

```diff
@@ -92,19 +92,33 @@ export class ResizeGroup extends BaseComponent<IResizeGroupProps, IResizeGroupSt
   }
 
   private _onResize() {
-    // If we have some cached measurements, let's see if we can skip rendering
+    let shouldMeasure = true;
```

Comment:
It's to make sure we don't set state in the event that we don't need to remeasure. It was cleaner to do this instead of a bunch of early returns